CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.1.2]
```

Comment:
* I wouldn't use came case in change log  
* Also check the changes linking (the links are defined at the end of the change log file)

Reply:
Fixed, thanks